src/styles.scss

```diff
@@ -1000,6 +1000,13 @@ mat-tab-group.no-pagination {
   color: rgba(0, 0, 0, 0.54);
 }
 
+// Styling for information below the user icon in user-pages
+.mat-card-user-info {
```

Comment:
I think this file should have global styles that might be used in multiple places. This seems sort of specific to the one page. Having a class that encapsulates `text-align: center;` seems like a good candidate, which I think I was Steve's original point.

On the other hand, if you envision this being used in other places, then it seems OK. I think I'll defer to Kathy on this, who has a better sense of our styling.

Reply:
Ohh okay that makes sense to me, I'll make the changes :+1: thank you for the suggestion and clarification!